lib/checks/keyboard/landmark-is-top-level.js

```diff
@@ -0,0 +1,20 @@
+const landmarks = axe.commons.aria.getRolesByType('landmark');
+const sectioning = ['article', 'aside', 'main', 'navigation', 'section'];
+const nodeIsHeader = node.tagName.toLowerCase() === 'header' && node.getAttribute('role') !== 'banner';
+const nodeIsFooter = node.tagName.toLowerCase() === 'footer' && node.getAttribute('role') !== 'contentinfo';
+var parent = axe.commons.dom.getComposedParent(node);
+
+while (parent){
+	var role = parent.getAttribute('role');
```

Comment:
Naming this `parentRole` would make it more clear what's being evaluated, since we just looked at the node's role above.

Reply:
Updated the code to use parentNode